safe/gui/tools/test/test_multi_buffer.py

```diff
@@ -77,6 +72,14 @@ def multi_buffer_test(self, output_path):
 
         self.assertEqual(expected_fields_name, new_field_names)
 
+        # We need to clean generated files
+        try:
+            os.remove(self.output_path)
+            os.remove(self.output_path.replace('.geojson', '.xml'))
```

Comment:
if the first one raise exception, the second one wouldn't be run. Is that ok?

Reply:
yes it's ok, because the XML is linked to the geojson ;-)